File: solutions/python/difference-of-squares/1/difference_of_squares.py
def square_of_sum(number):
    sum = 0
    for i in range(1,number+1):
        # print(i)
        sum = sum + i
    return sum**2
        
def sum_of_squares(number):
    sum_sqr = 0
    for j in range(1,number+1):
        # print(j)
        sum_sqr = sum_sqr + j**2
    return sum_sqr


def difference_of_squares(number):
    return square_of_sum(number)-sum_of_squares(number)

File: solutions/c/difference-of-squares/1/difference_of_squares.c
#include "difference_of_squares.h"
#include <stdio.h>

unsigned int sum_of_squares(unsigned int number)
{
    int sum_of_square = 0;
    for (unsigned int i = 1; i <= number; i++)
    {
        sum_of_square += i*i;
    }
    printf("%i", sum_of_square);
    return sum_of_square;
}
unsigned int square_of_sum(unsigned int number)
{
    int square_of_sum = 0;
    for(unsigned int i = 1; i <= number; i++)
    {
        square_of_sum += i; 
    }
    printf("%i", square_of_sum);
    printf("%i", square_of_sum * square_of_sum);
    return square_of_sum * square_of_sum;
}
unsigned int difference_of_squares(unsigned int number)
{ 
    printf("%i", square_of_sum(number) - sum_of_squares(number));
    return square_of_sum(number) - sum_of_squares(number);
}